rust/Cargo.toml

```diff
@@ -52,6 +52,8 @@ rand = "0.8.5"
 redis = "0.22.3"
 regex = "1.10.6"
 ripemd = "0.1.3"
+russh = "0.45.0"
+russh-keys = "0.45.0"
```

Comment:
Having these here even if compiling with `nasl-builtin-libssh` is a little unfortunate, but the alternative is to

1. Add a feature `nasl-builtin-russh`.
2. Activate that feature by default
3. Toggle the `russh` dependency on that feature.

but then specifying `--features experimental` will not work directly, since it would not actually turn the `nasl-builtin-russh` feature off, so we'd have to do `--no-default-features --features experimental,and,all,other,features,that,are,in,default` which is very annoying.